src/graphql/main/schema.graphql

```diff
@@ -877,6 +877,8 @@ scalar PaymentHash
 
 type PaymentSendPayload {
   errors: [Error!]!
+  transaction: Transaction
+  wallet: Wallet
```

Comment:
Since the wallet object is changing (the balance is different and the history) as a result of this mutation, it would be helpful to return the Wallet object and have the cache be automatically updated.

Reply:
wallet. mandatory?